src/main/kotlin/com/nexters/gamss/llm/generation/GeminiEmotionExtractor.kt

```
@@ -0,0 +1,122 @@
+                ?: throw CardGenerationFailedException(
+                    "카드 감정 분류 응답이 비어 있습니다.",
+                    usedTokens = usedTokens,
+                    cachedTokens = cachedTokens,
+                    inputTokens = inputTokens,
+                    outputTokens = outputTokens,
+                )
+
+        val emotion =
+            try {
+                parseEmotion(text)
+            } catch (e: CardGenerationFailedException) {
```

detekt (1.23.8)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)